### PR TITLE
[RDY] Updating Grafana to  6.3.0-beta1

### PR DESCRIPTION
**Pull request by geekodour**
This PR updates Grafana to 6.2.5 and performs the migration steps ( https://grafana.com/docs/installation/docker/#migration-from-a-previous-version-of-the-docker-container-to-5-1-or-later ) ~few things still need to be tested.~

### CODE REVIEW

File: manifests/cluster-infra/grafana_datasource.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ data:
       type: prometheus
       access: proxy
       orgId: 1
-      url: http://{{ .DOMAIN_NAME }}/prometheus-meta/
+      url: http://prometheus-meta/prometheus-meta/
```
Comment (krasi-georgiev) [inline manifests/cluster-infra/grafana_datasource.yaml:19]: Why this change? can't we use the env var here as well?

Comment (geekodour) [reply manifests/cluster-infra/grafana_datasource.yaml:19]: this will allow grafana to source prometheus even if the domain is not pointed at it. let me know if I should revert it back to using `.DOMAIN_NAME`

Comment (krasi-georgiev) [reply manifests/cluster-infra/grafana_datasource.yaml:19]: I think this is less likely to brake if we change the name of the service at some point.

Comment (krasi-georgiev) [reply manifests/cluster-infra/grafana_datasource.yaml:19]: or maybe put in some global env variable so that if we change the service name it will update everywhere.

**Comment by krasi-georgiev** [comment]
btw grafana 6.3 should be released next week so might be worth to wait for it.

**Comment by geekodour** [comment]
Yes, It'll be good to wait for it.

File: manifests/cluster-infra/grafana_datasource.yaml (hunk 1 above)
Comment (geekodour) [reply manifests/cluster-infra/grafana_datasource.yaml:19]: I'll revert this change and add the change you mentioned in another PR.

Comment (geekodour) [reply manifests/cluster-infra/grafana_datasource.yaml:19]: this change is required for changes in #227 since the config files don't take any variables.

https://github.com/prometheus/prombench/blob/7a2c225bff4ef50629867aeeebae419f585c4449/config/grafana/prometheus-datasource.yaml

**Comment by geekodour** [comment]
updated to  6.3.0-beta1 

cc @krasi-georgiev

**Comment by krasi-georgiev** [comment]
Thanks
